src/App.js

```diff
@@ -1,22 +1,229 @@
VhY3QtY29udGV4dC1wcm92aWRlciIsImxhc3RfbGVzc29uX2NvbXBsZXRlZF9hdCI6IjIwMTctMTEtMzBUMTI6NTI6MjIiLCJ0b3RhbCI6MC4wLCJ0cmFuc2FjdGlvbnMiOjAsIm1lbWJlcl9zaW5jZSI6IjIwMTUtMTItMTRUMTA6MTQ6MDMiLCJpbnRlcnZhbCI6Im5ldmVyIiwicGxhbl9uYW1lIjoiTWFuYWdlZCBTdWJzY3JpcHRpb24iLCJjdXJyZW50X2NvdXJzZV90aXRsZSI6IlN0cnVjdHVyZSBBbmd1bGFyIEFwcHMgd2l0aCBBbmd1bGFyIE1hdGVyaWFsIENvbXBvbmVudHMiLCJjdXJyZW50X2NvdXJzZV9kZXNjcmlwdGlvbiI6IkluIHRoaXMgY291cnNlIHdlIHdpbGwgZXhwbG9yZSBpbnRlZ3JhbCBBbmd1bGFyIE1hdGVyaWFsIGNvbXBvbmVudHMuIFdlIHdpbGwgc3RhcnQgd2l0aCBjb21wb25lbnRzIHJlbGF0ZWQgdG8gbmF2aWdhdGlvbiwgbGF5b3V0LCBhbmQgc3RydWN0dXJlLiBXZSB3aWxsIHRoZW4gbW92ZSB0aHJvdWdoIHRoZSB1dGlsaXR5LCBpbnB1dCwgYW5kIGRhdGF0YWJsZSBjb21wb25lbnRzLCBkZW1vbnN0cmF0aW5nIGhvdyBlYWNoIGNhbiBiZSB1c2VkIHRvIHN1aXQgeW91ciBwcm9qZWN04oCZcyBuZWVkcy4gRmluYWxseSB3ZSB3aWxsIHRvdWNoIG9uIHRoZW1pbmcgeW91ciBhcHBsaWNhdGlvbiB0byBjcmVhdGUgYSBjdXN0b20gbG9vayBhbmQgZmVlbCB0aGF0IGZpdHMgd2l0aGluIHRoZSBNYXRlcmlhbCBEZXNpZ24gZ3VpZGVsaW5lcy5cbkJ5IHRoZSBlbmQgb2YgdGhlIGNvdXJzZSB5b3Ugd2lsbCBiZSBjb21mb3J0YWJsZSBsZXZlcmFnaW5nIHRoZSBBbmd1bGFyIE1hdGVyaWFsIGxpYnJhcnkgdG8gY3JlYXRlIGFuIG91dHN0YW5kaW5nIGludGVyZmFjZSBhbmQgdXNlciBleHBlcmllbmNlIVxuIiwiY3VycmVudF9jb3Vyc2VfdXJsIjoiaHR0cHM6Ly9lZ2doZWFkLmlvL2NvdXJzZXMvc3RydWN0dXJlLWFuZ3VsYXItYXBwcy13aXRoLWFuZ3VsYXItbWF0ZXJpYWwtY29tcG9uZW50cyIsImN1cnJlbnRfY291cnNlX3RlY2hub2xvZ3kiOiJBbmd1bGFyIDIiLCJjdXJyZW50X2NvdXJzZV9pc19jb21wbGV0ZSI6ZmFsc2UsImN1cnJlbnRfY291cnNlX2NvbXBsZXRlZF9sZXNzb25fY291bnQiOjAsImN1cnJlbnRfY291cnNlX3N0YXJ0ZWRfb24iOiIyMDE4LTAxLTMxVDE1OjAyOjQ4IiwiY3VycmVudF9jb3Vyc2VfbGFzdF93YXRjaGVkX29uIjpudWxsLCJjdXJyZW50X2NvdXJzZV9uZXh0X2xlc3Nvbl90aXRsZSI6IkFuZ3VsYXIgTWF0ZXJpYWwgQ29tcG9uZW50cyBDb3Vyc2UgT3ZlcnZpZXciLCJjdXJyZW50X2NvdXJzZV9uZXh0X2xlc3Nvbl9zdW1tYXJ5IjoiSW4gdGhpcyBjb3Vyc2Ugd2UnbGwgc3RhcnQgZnJvbSBzY3JhdGNoIGJ5IGluc3RhbGxpbmcgYW5kIGNvbmZpZ3VyaW5nIEFuZ3VsYXIgTWF0ZXJpYWwgdG8gYmUgdXNlZCBpbiBhIHByb2plY3QuIFRoaXMgd2lsbCBpbmNsdWRlIGNvbmZpZ3VyaW5nIGEgY3VzdG9tIEFuZ3VsYXIgTWF0ZXJpYWwgbW9kdWxlIGFzIHdlbGwgYXMgc29tZSBwcmVidWlsdCB0aGVtZXMgYXZhaWxhYmxlIHRvIHVzLlxuV2UnbGwgbG9vayB0aHJvdWdoIHRoZSBNYXRlcmlhbCBhcGkgYXMgd2VsbCBhcyBhcHBseSB0aXBzIGFuZCB0cmlja3Mgb24gY29tbW9uIHByb2JsZW1zIHlvdSdsbCBmYWNlIHdoZW4gYXBwbHlpbmcgdGhpcyBzdHlsZSB0byB5b3VyIGFwcGxpY2F0aW9uLiBXZSdsbCBjb3ZlciB0b3BpY3Mgc3VjaCBhcyBob3cgTWF0ZXJpYWwgaGFuZGxlcyB1c2VyIGlucHV0LCBjb21tb24gTWF0ZXJpYWwgdXRpbGl0eSBjb21wb25lbnRzIChzcGlubmVycywgcHJvZ3Jlc3MgYmFycyBldGMuKSwgYW5kIGhvdyB0byBwcm9wZXJseSBtYW5hZ2UgdGFidWxhciBkYXRhIGFuZCB0aGVtZXMgaW4gQW5ndWxhciBNYXRlcmlhbC5cbkJ5IHRoZSBlbmQgb2YgdGhpcyBjb3Vyc2UsIHlvdSB3aWxsIGJlIGNvbWZvcnRhYmxlIGxldmVyYWdpbmcgdGhlIEFuZ3VsYXIgTWF0ZXJpYWwgY29tcG9uZW50IGxpYnJhcnkgdG8gY3JlYXRlIGFwcGxpY2F0aW9ucyB0aGF0IGxvb2sgYW5kIGZlZWwgZ3JlYXQgYmFzZWQgb24gdGhlIE1hdGVyaWFsIGRlc2lnbiBndWlkZWxpbmVzIGFuZCBoYXZlIGEgaGlnaCBmb2N1cyBvbiBhY2Nlc3NpYmlsaXR5LlxuIiwiY3VycmVudF9jb3Vyc2VfbmV4dF9sZXNzb25fdXJsIjoiaHR0cHM6Ly9lZ2doZWFkLmlvL2xlc3NvbnMvYW5ndWxhci1hbmd1bGFyLW1hdGVyaWFsLWNvbXBvbmVudHMtY291cnNlLW92ZXJ2aWV3In19.5dJgAm43vwS5Xh6I-D-kpY_b_J-kREI6srMMj5WJt5M';
```

Comment:
http://egghead.af:5000/users/jwt?return_to=http://localhost:3000 will give you a token